catalog/kubedb/active_versions.json

```diff
@@ -51,12 +51,12 @@
     "1.18.0"
   ],
   "Kafka": [
+    "3.9.0",
+    "3.8.1",
+    "3.7.2",
     "3.6.1",
     "3.6.0",
```

Comment:
I think we should deprecate 3.6.0 too. Since it has a major bug. That's why we added 3.6.1 later.